src/pta_export/core/document.py

```diff
@@ -208,7 +208,10 @@ def add_header(document: Document, vak: Vak, year: int, leerjaar: int):
     if vak.afkorting and vak.weergeven == 1:
         vak_naam = f"{vak_naam} ({vak.afkorting})"
 
-    header_run = header_p.add_run(f"PTA\t{vak_naam}\t{_leerjaar}\t{school_year}")
+    if leerjaar == 16:
+        header_run = header_p.add_run(f"PTA\t{vak_naam}\t{_leerjaar}\t{school_year}")
+        else:
```

Comment:
this one should not be indented along, but instead be:

```suggestion
    else:
```

I'll make a couple tweaks to the PR and then merge it & deploy the changes.